MdeModulePkg/Library/DxeCapsuleLibFmp/DxeCapsuleLib.c

```diff
@@ -1402,6 +1405,16 @@ IsNestedFmpCapsule (
       }
     }
   } else {
+    if (mDxeCapsuleLibReadyToBootEvent) {

```

Comment:
The last time gBS->LocateProtocol() can be called is ExitBootServices().  Are there cases where this API is used between ReadyToBoot and ExitBootServices?

Reply:
I don't see the case that gBS->LocateProtocol () is called at runtime. Could you introduce it?

Reply:
The `mEsrtTable` is NULL if the ESRT is not located at ReadyToBoot. During runtime, when `IsNestedFmpCapsule()` is called, it defaults to the 'else' case, resulting in a crash.

Reply:
I agree this problem. This fix is good.